apps/server/src/core/logger/default.loggable.ts

```diff
@@ -0,0 +1,30 @@
+import { BaseEntity, EntityId } from '@shared/domain';
+
+type LogDataType = BaseEntity | EntityId;
+
+// if we wanted a generic Loggable, it could look somewhat like this. But I, Thomas, dont think we should have one.
+export class DefaultLoggable {
+	data: { [key: string]: LogDataType };
+
+	message: string;
+
+	constructor(message: string, data: { [key: string]: LogDataType }) {
+		this.message = message;
+		this.data = data;
+	}
+
+	getLogMessage() {
+		const result = {
+			message: this.message,
+			data: {},
+		};
+		Object.entries(this.data).forEach(([key, value]) => {
+			if (value instanceof BaseEntity) {
+				result.data[key] = value.id;
+			}
+			result.data[key] = value;
```

Comment:
I don't really get this part. After result.data[key] is set to value.id it is instantly set to value. So why set it to value.id first?

Reply:
check if this class is needed at all. I would rather get rid of it.
Think of it as a draft in case we need a fallback loggable that can handle literally any thing that might come along.
If we need it, we also need to think about how to secure it to possibly prevent sensitive data from being logged. The part `if (value instanceof BaseEntity) {result.data[key] = value.id;}` does this exemploratory for entities, but there may be other kinds of data in there that right now would not be filtered or secured in any way.

the line you pointed out a mistake. should be an else clause.